client/components/Button/Button.js

```diff
@@ -3,14 +3,16 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import styles from './Button.css';
 
-const Button = ({ onClick, children, theme }) => {
+const Button = ({ onClick, children, theme, isDisabled }) => {
   return (
     <button
       className={classNames({
         [styles.root]: true,
         [styles[theme]]: true,
+        [styles.disabled]: isDisabled,
```

Comment:
I think we don't need the disabled class as we can target [disabled] in css right away.

Reply:
char